cmd/podman/volume_create.go

```diff
@@ -28,7 +28,8 @@ be created at.`
 		},
 		Example: `podman volume create myvol
   podman volume create
-  podman volume create --label foo=bar myvol`,
+  podman volume create --label foo=bar myvol
+  podman volume create --opt type=nfs --opt o=addr=192.168.0.2,rw --opt device=:/nfsshare  mynfsvol`,
```

Comment:
I've been trying to keep these to three examples.   Could we kill line 31 and add '--label foo=bar' to line 29?

Reply:
Fixed